### ISSUE TITLE
[1.9.2] deploying to heroku slug size over 500 MB

### ISSUE THREAD

**Issue by jamalcoder** [meteor/meteor#10938]
hello
when i updated to meteor 1.9.2 i can't deploy to heroku. always the slug size is higher then 500 MB
i tried  with different accounts and initialized my git repo  also i cleaned the heroku repo and cache but no success.

**Comment by hexsprite** [comment]
one idea is to use the bundle visualizer to see what's taking up space in your bundle. I'm not sure how much larger the core dependencies are from previous versions to 1.9.2 but at least the bundle is something you have more control over...

**Comment by flean** [comment]
Something is wrong with 1.9.2

My deploy went from 119 to 330

galaxy is also failing for me to deploy

**Comment by jamalcoder** [comment]
hi @hexsprite 1.91 work fine it's has nothing to do with components bundle.
hi @flean  me too i think 1.9.2 has an inssue.

**Comment by flean** [comment]
Agreed , I just pushed 1.9.1 and had no issues

**Comment by hexsprite** [comment]
using a disk space exploring tool can you see which files / directories have become larger? this will help determine where the problem is.

On Mac you can use Disk Inventory  X (http://www.derlien.com/) or command line utility du works in a pinch. Something similar for windows I'm sure...

**Comment by filipenevola** [comment]
Hi @jamalcoder and @flean could you share which folders/modules are taking more space?

I have created a new app with 1.9.1 and another with 1.9.2 and both have very similar bundle sizes after Meteor build, see:
```
-rw-r--r--   1 filipe  staff  18538808 Feb 25 11:23 app-192.tar.gz
-rw-r--r--   1 filipe  staff  18538263 Feb 25 11:23 app-191.tar.gz
```
We need more information to understand what is happening in your cases.

**Comment by filipenevola** [comment]
@SimonSimCity could you help here? I think the last changes in the roles package were made by you.

**Comment by jamalcoder** [comment]
downgrading alanning:roles version from 3.2.1 to 3.2.0 and this issue has gone i don't know why alanning:roles 3.2.1 cause this issue...is it a memory meak ? maybe...

**Comment by filipenevola** [comment]
alanning:roles@3.2.1_1 fixes this issue. 

The problem in 3.2.1 was the npm dev dependencies were published with the package.

**Comment by SimonSimCity** [comment]
My apologises ... I've published the package with all the dev dependencies - including a full installation of chrome (as part of the dev package `puppeteer`). Should be fixed in `v3.2.1_1`.

Please close if this was all which was unexpectedly large.
